app/services/problem_check/bad_favicon_url.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ProblemCheck::BadFaviconUrl < ProblemCheck
+  self.priority = "low"
+
+  def call
```

Comment:
> I wonder if we could add a subclass to indicate this better

Absolutely agree. This would add some important information about what this is. 👍 

> I think the other alternative would be to move the logic for the checks inside the actual problem check classes, but in a way they can be called from the outside

I think this doesn't quite fit with the problem that needs to be solve here, which is some random code path "stumbled upon" a problem, not as part of a problem check, but just part of normal execution. Maybe `ProblemCheck` should actually just be `Problem`, but I think we go with the `InlineProblemCheck` subclass first, then see where the design pressure takes us from there.